content/modules/redisearch/redisearch-quickstart.md

```diff
@@ -16,4 +16,10 @@ For this quick start tutorial, you need:
     - A Redis Enterprise Software database [with the RediSearch module]({{< relref "/modules/add-module-to-database.md" >}})
 - redis-cli with connectivity to a redis database
 
+## Try out RediSearch 2.x
+
+{{< embed-md "tryout-redisearch-2.md" >}}
+
+## Try out RediSearch 1.x
```

Comment:
why do we still have this?  I wouldn't confuse users and move all of them to 2.x asap